disassemblers/ofrak_ghidra/ofrak_ghidra/components/ghidra_analyzer.py

```diff
@@ -223,3 +227,46 @@ def _build_ghidra_server_args(self) -> List[str]:
             args.append(ghidra_script.split(".")[0].lower())
 
         return args
+
+
+@dataclass
+class GhidraCodeRegionConfig(ComponentConfig):
+    pass
+
+
+class GhidraCodeRegionAnalyzer(Analyzer[Optional[GhidraCodeRegionConfig], CodeRegion], OfrakGhidraMixin):
```

Comment:
This actually acts as a `Modifier`, changing the definition to be that of a `Modifier` will I think make this ready for merge